gm4_mending_tanks/beet.yaml

```diff
@@ -12,7 +12,7 @@ meta:
   gm4:
     versioning:
       required:
-        gm4_liquid_tanks: 2.7.0
+        gm4_liquid_tanks: 2.8.0
```

Comment:
Since the old mending tanks version broke together with this new liquid tanks version, should this not count as a major (breaking) change, and thus warrant `3.0.0`?

Reply:
That seems like a breaking change to me